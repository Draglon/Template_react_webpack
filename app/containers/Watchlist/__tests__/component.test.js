import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import WatchlistComponent from '../component';

describe('<WatchlistComponent /> container', () => {
  let component;

  it('render property', () => {
    component = shallow(<WatchlistComponent />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
