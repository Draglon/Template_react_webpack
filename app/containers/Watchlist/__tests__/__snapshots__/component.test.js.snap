// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WatchlistComponent /> container render property 1`] = `
<Fragment>
  <h1>
    Watchlist
  </h1>
</Fragment>
`;
